### PR TITLE
About us

### PR DESCRIPTION
**Pull request by Lameya56**

### CODE REVIEW

File: views/js/evently/src/RSVPForm.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
-* {font-family: "Bree Serif";}
+/* * {font-family: "Bree Serif";} */
```
Comment (mclausen792) [inline views/js/evently/src/RSVPForm.css:1]: Does this mess up the font on the RSVP form now?

**Review by mclausen792 — COMMENTED** [review]
Seems pretty straight forward. Have a question about possible conflicts

File: views/js/evently/src/RSVPForm.css (hunk 1 above)
Comment (Lameya56) [reply views/js/evently/src/RSVPForm.css:1]: It looks fine on my end

Comment (Lameya56) [reply views/js/evently/src/RSVPForm.css:1]: I think Iden fixed the bree-serif font issue in another pull request

**Review by dianaabishop — APPROVED** [review]

**Pull request closed without merging**